ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -416,6 +418,16 @@ public struct Avatar: View, TokenizedControlView {
                                 dark: GlobalTokens.sharedColors(colorSet, .tint40))
         }
 
+        static func ringColor(fromPrimaryText primaryText: String?,
+                              secondaryText: String?,
+                              fluentTheme: FluentTheme) -> DynamicColor {
```

Comment:
dumb question : why do we need fluentTheme?

Reply:
I assume it was needed to access `GlobalTokens` before @mischreiber's `GlobalTokens` changes. I'll go ahead and remove it.